docker/node/server.js

```diff
@@ -197,6 +197,75 @@ LinkedList.Circular.prototype.toArray = function () {
 	return out;
 };
 
+
+LinkedList.Circular.prototype.some = function(cb) {
+	return this.find(cb) !== null;
+}
+
+LinkedList.Circular.prototype.find = function(cb) {
+	let video = this.first;
+
+	for (let i = 0; i < this.length; i++) {
+		if (cb(video, i)) {
+			return video;
+		}
+
+		video = video.next;
+	}
+
+	return null;
+}
+LinkedList.Circular.prototype.multiple = function(indexes) {
```

Comment:
I would suggest making sure that this method has a verb in the name (unless this name comes from a standard somewhere?). It's fairly clear what it does at the callsites, but it took a second to figure out what it did when I first saw this line. Something like `getMultipleByIndex` or something.

super-nitpicky optimization suggestions:
1. construct the result array in the loop to remove the last line:
```js
const result = [];
for (...) {
  if (stuff) {
    result.push(...);
  }
}
return result;
```
2. use a `Set` instead of `Map` to save a few array allocations
3. calculate the max index at the same time as populating the `Set`